### DISCUSSION TITLE
litd command not found

### DISCUSSION THREAD

**Discussion by cmyk** [lightninglabs/lightning-terminal#711] [General]
I followed the instructions here to the point: https://docs.lightning.engineering/lightning-network-tools/lightning-terminal/get-lit
Compiling everything went fine.
But I don't have the command litd. There's no binary in /usr/local/bin nor /bin/

**Comment by cmyk** [comment]
The binaries have been created in my home folder under ~/go/bin
No idea why.

**Reply by guggero** [reply]
Can you check what our `$GOPATH` variable points to?

**Comment by cmyk** [comment]
No idea how to.
`echo $GOPATH` just prints an empty line.

**Reply by guggero** [reply]
And what about `go env GOPATH`? The `litd` binary should be somewhere below that path (assuming you didn't get any errors when running the installation).

**Comment by cmyk** [comment]
/home/cmyk/go

**Reply by guggero** [reply]
Then something went wrong when building. Can you please share the full command you used and all output generated by it?
